updates/location_asset_mappings/v2.json

```diff
@@ -0,0 +1,122 @@
+{
+    "location_asset_mappings": {
+      "additions": [
+        {
+          "asset": "eip155:1/erc20:0x5aFE3855358E112B5647B952709E6165e1c1eEEe",
+          "location": "coinbase",
+          "location_symbol": "SAFE"
+        },
+        {
+            "asset": "eip155:56/erc20:0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+            "location": "binance",
+            "location_symbol": "CAKE"
+        },
+        {
+            "asset": "eip155:56/erc20:0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+            "location": "binanceus",
+            "location_symbol": "CAKE"
+        },
+        {
+            "asset": "eip155:1/erc20:0x576e2BeD8F7b46D34016198911Cdf9886f78bea7",
+            "location": "poloniex",
+            "location_symbol": "TRUMP"
+        },
+        {
+            "asset": "eip155:1/erc20:0xB4EFd85c19999D84251304bDA99E90B92300Bd93",
+            "location": "okx",
+            "location_symbol": "RPL"
+        },
+        {
+            "asset": "eip155:1/erc20:0x5aFE3855358E112B5647B952709E6165e1c1eEEe",
+            "location": "okx",
+            "location_symbol": "SAFE"
+        },
+        {
+            "asset": "eip155:1/erc20:0x46F84DC6564Cdd93922F7BFb88B03D35308D87C9",
+            "location": "okx",
+            "location_symbol": "VENOM"
+        },
+        {
+            "asset": "eip155:1/erc20:0x91Af0fBB28ABA7E31403Cb457106Ce79397FD4E6",
+            "location": "okx",
+            "location_symbol": "AERGO"
+        },
+        {
+            "asset": "eip155:1/erc20:0xf091867EC603A6628eD83D274E835539D82e9cc8",
+            "location": "kraken",
+            "location_symbol": "ZETA"
+        },
+        {
+            "asset": "eip155:1/erc20:0x33349B282065b0284d756F0577FB39c158F935e6",
+            "location": "kraken",
+            "location_symbol": "MPL"
+        },
+        {
+            "asset": "eip155:1/erc20:0xB528edBef013aff855ac3c50b381f253aF13b997",
+            "location": "kraken",
+            "location_symbol": "AEVO"
+        },
+        {
+            "asset": "eip155:1/erc20:0x0D3CbED3f69EE050668ADF3D9Ea57241cBa33A2B",
+            "location": "kraken",
+            "location_symbol": "PDA"
+        },
+        {
+            "asset": "eip155:1/erc20:0x5aFE3855358E112B5647B952709E6165e1c1eEEe",
+            "location": "kraken",
+            "location_symbol": "SAFE"
+        },
+        {
+            "asset": "TREMP",
+            "location": "kraken",
+            "location_symbol": "TREMP"
+        },
+        {
+            "asset": "BODEN",
+            "location": "kraken",
+            "location_symbol": "BODEN"
+        },
+        {
+            "asset": "TNSR",
+            "location": "kraken",
+            "location_symbol": "TNSR"
```

Comment:
when the asset is the same as location_symbol, then it should not be needed because we fallback to the same value as default for all the exchanges.
```py
        name = GlobalDBHandler.get_assetid_from_exchange_name(
            exchange=Location.KRAKEN,
            symbol=kraken_name,
            default=kraken_name,
        )
    return symbol_to_asset_or_token(name)
```
tradeoff is that it increases json's size, but can probably make it future safe if we ever change that logic

Reply:
Kraken needed them always before, not sure if that changed recently. Updated it